core/errors/packager.h

```diff
@@ -21,5 +21,7 @@ constexpr ErrorClass DefinitionPackageMismatch{3713, StrictLevel::False};
 constexpr ErrorClass ImportConflict{3714, StrictLevel::False};
 constexpr ErrorClass InvalidExportForTest{3715, StrictLevel::False};
 constexpr ErrorClass ExportConflict{3716, StrictLevel::False};
+constexpr ErrorClass InvalidTypedLevelDeclaration{3717, StrictLevel::False};
```

Comment:
```suggestion
constexpr ErrorClass InvalidMinTypedLevelDeclaration{3717, StrictLevel::False};
```

Is this name more accurate?